tests/UnitTests.php

```diff
@@ -200,6 +200,28 @@ public function testWhitelistSettingShortcutWithMultipleRanges()
 
         $this->assertEquals($expected, $actual);
     }
+    
+    /**
+     * Spaces on executable path management (Windows, Linux) :
+     */
+    public function testPathExecutable()
+    {
+        $expected = '"C:\Program Files\Tesseract OCR\tesseract.exe"';
+
+        // Windows : unescaped path
+        $actual1 = (new WrapTesseractOCR())
+            ->executable("C:\Program Files\Tesseract OCR\tesseract.exe");
+        // Windows : escaped path
+        $actual2 = (new WrapTesseractOCR())
+            ->executable('"C:\Program Files\Tesseract OCR\tesseract.exe"');
+        // Linux
+        $actual2 = (new WrapTesseractOCR())
```

Comment:
isn't it supposed to be `$actual3` ?
anyways, i'm not willing to accept this PR regardless of this problem. tests shouldn't have multiple assertions, the implementation should be OS agnostic and i'd prefer a solution without IF statements. (https://cirillocompany.de/pages/anti-if-campaign)

I'm glad you took the time to contribute, but if the PR isn't aligned with my values I'd rather not merge them.
I'd encourage you to fork it, so that you can still benefit from the code, and will be free to steer it in different directions, without an annoying guy declining your pull request ;D